UniversityGear/.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -0,0 +1,10 @@
+<component name="InspectionProjectProfileManager">
```

Comment:
I'll recommend to not track this file with Git, try to add a gitignore and remove this metadata IDE files from the repo.